service/api_patch.go

```diff
@@ -191,6 +193,16 @@ func (as *APIServer) updatePatchModule(w http.ResponseWriter, r *http.Request) {
 		as.LoggedError(w, r, http.StatusInternalServerError, err)
 		return
 	}
+	if formatted {
+		// refresh the reader
+		reader := strings.NewReader(patchContent)
+		summariesByCommit, err := units.GetPatchSummariesByCommit(reader)
+		if err != nil {
+			as.LoggedError(w, r, http.StatusInternalServerError, errors.Errorf("Error parsing formatted commit"))
+			summaries = summariesByCommit
```

Comment:
I would think we'd want this when `err == nil`

Reply:
true